linode/manager
# fix: [UIE-8152] - DBaaS V2 Resize: disable Shared plans tab

## Description 📝
Resize Database Cluster: disable 'Shared CPU' tab for 2-nodes Database Cluster.

## Changes  🔄
List any change relevant to the reviewer.
- disable 'Shared CPU' tab in Database Resize Plans Table
- add tooltip for disabled tab

## Target release date 🗓️
9/30/24

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-09-24 at 4 58 05 PM](https://github.com/user-attachments/assets/28197986-f6b0-4036-803b-8d2e5eaa26d9) | ![Screenshot 2024-09-24 at 4 58 37 PM](https://github.com/user-attachments/assets/5f5a7eef-dd23-4c7d-b427-eaa31058f418)|
| | ![Screenshot 2024-09-24 at 4 58 52 PM](https://github.com/user-attachments/assets/885d7042-5e24-4395-8963-20e77dc5132b) |

## How to test 🧪

### Prerequisites
(How to setup test environment)
**Enable DatabaseV2Beta flag**

### Verification steps
(How to verify changes)

- Create a Database Cluster with 2 nodes.
- Go to the created Database Cluster.
- Navigate to the Resize tab.
- Shared CPU tab is disabled and displays a tooltip.

## As an Author I have considered 🤔

*Check all that apply*
